extras/rya.export/README.md

```diff
@@ -0,0 +1,250 @@
+| 6x       | newer      | newer     | same key        | do nothing             | yes                 |
+| 7\*      | older      | newer     | child updated   | do cases 1 and 4       | yes                 |
+| 8\*      | newer      | older     | parent updated  | do cases 2 and 3       | yes                 |
+
+NOTE:
+
+**x** - The two cases having the same key and timestamp are already merged.
+
+**\*** - The last two cases are really two distinct keys, since the timestamp is part of the Accumulo key. They are handled as two comparisons. They are different versions of the same key, but Rya is generally used with only the latest version.
+
+Here it is again, in full English:
+
+1.  If the parent statement exists, child doesn't, and parent time-stamp is before the start time, assume child statement was deleted, so delete the parent statement.
+
+2.  If the parent statement exists, child doesn't, and parent time-stamp is after the start time, assume parent statement was added, so do nothing.
+
+3.  If the child statement exists, parent doesn't, and child time-stamp is before the start time, assume parent statement was deleted, so do nothing.
+
+4.  If the child statement exists, parent doesn't, and child time-stamp is after the start time, assume child statement was added, so add the statement to the parent.
+
+## Instructions
+
+### Prerequisites
+
+If one or more of the Rya instance datastore uses Accumulo, then Accumulo and Hadoop will be needed. These can skipped if a Accumulo instance already exists for Rya to connect to. These directions are mostly for running on a local machine. Likewise, MongoDB can be setup to run locally as well but isn't necessary if connecting to an existing MongoDB instance.
+
+If you are building on Windows, you will need hadoop-common 2.6.0's *"winutils.exe"* and *"hadoop.dll"*. You can download it from [here](https://github.com/amihalik/hadoop-common-2.6.0-bin/archive/master.zip).
+
+Environment variables for `ACCUMULO_HOME` and `HADOOP_HOME` need to be set.
+
+#### Accumulo Home variable:
+
+You'll need to set an environment variable for `ACCUMULO_HOME`. (at least for Windows) It does not need to exist, but it must be an absolute path.
+
+`set ACCUMULO_HOME=C:\accumulo-1.6.4`
+
+It uses this to find extra jar's: "$ACCUMULO\_HOME/lib/ext/\[^.\].\*.jar" If this variable is empty, or if it is a relative path, it will produce this error:
+
+`Could not find file with URI "/lib/ext/[^.].*.jar" because it is a relative path, and no base URI was provided.`
+
+#### Hadoop Home variable:
+
+You'll need to set an environment variable for `HADOOP_HOME`.
+
+`set HADOOP_HOME=C:\hadoop-common-2.6.0-bin-master`  
+`set PATH=%PATH%;%HADOOP_HOME%\bin`
+
+#### MongoDB
+
+MongoDB can be downloaded at [https://www.mongodb.com/download-center](https://www.mongodb.com/download-center). Run the downloaded installer for it. After installing, create a `C:\data\db\` directory.
```

Comment:
Do we need instructions on how to setup the  various databases?  If so, this is also if using windows.  instead just replace C:\mongodb with  [Install Location]